app/cmd/plugin_search_test.go

```diff
@@ -44,10 +44,10 @@ var _ = Describe("plugin search command", func() {
 
 			request, _ := client.PrepareForEmptyAvaiablePluginList(roundTripper, "http://localhost:8080/jenkins")
 			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
-			request, _ = client.RequestUpdateCenter(roundTripper, "http://localhost:8080/jenkins")
-			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
-			request, _ = client.PrepareForOneInstalledPlugin(roundTripper, "http://localhost:8080/jenkins")
-			request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
+			// request, _ = client.RequestUpdateCenter(roundTripper, "http://localhost:8080/jenkins")
+			// request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
+			// request, _ = client.PrepareForOneInstalledPlugin(roundTripper, "http://localhost:8080/jenkins")
+			// request.SetBasicAuth("admin", "111e3a2f0231198855dceaff96f20540a9")
```

Comment:
If you don't need those code lines, please just remove them.

Reply:
I removed it